fixtures/flash-professional/led-moving-head-36x10w-zoom.json

```diff
@@ -0,0 +1,144 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "LED moving head 36x10w ZOOM",
```

Comment:
We might want to add the model number "F7100494" to the name, because there are some awfully similar fixtures from this company which have different DMX maps.